projects/ipdg_stokes/mesh/annulus_triag_mesh_builder.h

```diff
@@ -15,17 +15,21 @@ namespace mesh {
 
 /**
  * @brief A mesh builder for disks with a hole in the middle
+ *
+ * The diameter of the hole is controlled by the y-value passed to
+ * `setBottomLeftCorner(x, y)` while the diameter of the disk is controlled by
+ * the y-value passed to `setTopRightCorner(x, y)`.
```

Comment:
This interface is prescribed because the mesh builder inherits from `lf::mesh::hybrid2d::StructuredMeshBuilder`. This strategy is consistent with the approach used for the `lf::mesh::hybrid2d::TorusMeshBuilder`. Should I change this interface nonetheless or is the consistency with other parts of the code still more important?

Reply:
No this is not important at all. StructuredMeshBuilder is just a utility class that saves you some typing when making a new MeshBuilder, but it doesn't fit all use cases ;)